lib/index.js

```diff
@@ -124,3 +127,11 @@ function displayDateTime(){
 getWeatherButton.addEventListener("click", function(){
   processWeatherRequest();
 })
+
+favDropDownLink.addEventListener("click", function( event ) {
```

Comment:
This looks like the only place you are using `favDropDownLink`. If you plan on having to use it again, it is ok to make it a variable, otherwise, it is less confusing (for me) to have the `document.getElementById("dropDownFavs").addEventListener` here.